mss/check-redzone.cpp

```diff
@@ -0,0 +1,67 @@
+#include "include/mss.hpp"
+#include "include/assembly.hpp"
+
+long long redzone_class_var_stack,
+          redzone_class_var_heap,
+          redzone_class_var_data,
+          redzone_class_var_rodata;
```

Comment:
You dont need to define 4 variables here.
Each test is a single test that returns a length for a single condition!
Also, there is no need to use the long variable name here.
Long variable name is needed in the variable.json.
Inside each test, the meaning of variables are precise, aren't they?